app/views/parties/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= render partial: "./nav" %>
+
+<div class="page-container">
+  <div class="left-of-sidebar">
+    <h1><%= @party.name %></h1>
+    <p>Time: <%= @party.date.strftime("%b %d, %Y @ %I:%M %P") %></p>
```

Comment:
You can add this as a time format  that you could do party.date.to_form
In an initializer do 
```Time::DATE_FORMATS[:potluck] = "%b %d, %Y @ %I:%M %P"```
then in code this date could be ```@party.date.to_formatted_s(:potluck)```
thus giving you just one place to mange your app's date formats.